app/controllers/maintainers_controller.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+class MaintainersController < ApplicationController
+  before_action :load_project
+  def edit
+  end
+
+  def create
+    user = User.find_by(id: params[:maintainer_id])
+    @project.maintainers << user
```

Comment:
_disclaimer: nem fogom csak azért visszadobni, meg gondold végig, hogy neked is segítene-e ez a kódstílus. nem kötelező így csinálni, csak egy ötlet_

Én annyira nem szeretem az egyszer használatos változókat. Ha csak ennyi lenne az akcióban mint most, akkor én kispórolnám a `user = #...` értékadást.

Reply:
én inkább hibakezelést raknék a két sor közé: mi történik, ha nem létezik a user, akit hozzá szeretnénk kapcsolni?

Reply:
Igen igen. Erről lentebb van egy külön komment.